config/locales/eo.yml

```diff
@@ -7,30 +7,247 @@ eo:
     people_followed_by: Sekvatoj de %{name}
     people_who_follow: Sekvantoj de %{name}
     posts: Mesaĝoj
+    posts_with_replies: Tootoj kun respondaj
     remote_follow: Fore sekvi
+    reserved_username: La usantnomo estas reservis
+    roles:
+      admin: Administranto
     unfollow: Malsekvi
+  admin:
+    accounts:
+      are_you_sure: Ĉu vi certe?
+      confirm: Confirmi
+      confirmed: Confirmis
+      disable_two_factor_authentication: Malebligi 2FA
+      display_name: Montri nomo
+      domain: Domajno
+      edit: Redakti
+      email: Retpoŝto
+      followers: Sekvantoj
+      followers_url: Sekvantoj URL
+      follows: Sekvatoj
+      ip: IP
+      location:
+        all: Ĉio
+        local: Loka
+        remote: Fora
+        title: Loko
+      media_attachments: Komunkiloj kunsendaĵo
+      moderation:
+        all: Ĉio
+        silenced: Silentis
+        suspended: Suspendis
+        title: Moderulo
+      most_recent_activity: Ple freŝa aktiveco
+      most_recent_ip: Ple freŝa IP
+      not_subscribed: Ne abonis
+      order:
+        alphabetic: Alfabetiko
+        most_recent: Ple freŝa
+        title: Ordono
+      perform_full_suspension: Fari kompleta suspendi
+      profile_url: Profilo URL
+      protocol: Protokolo
+      public: Publika
+      push_subscription_expires: PuSH subscription expires
+      redownload: Refreŝigi avataro
+      reset: Restarigi
+      reset_password: Restarigi pasvorto
+      resubscribe: Reaboni
+      salmon_url: Salmon URL
+      search: Serĉi
+      shared_inbox_url: Shared Inbox URL
+      show:
+        created_reports: Raportoj kreita de ĉi tiu konto
+        report: raporto
+        targeted_reports: Raportoj kreita al ĉi tiu konton
+      silence: Silenti
+      statuses: Statusoj
+      subscribe: Aboni
+      title: Kontoj
+      undo_silenced: Malfari silenti
+      undo_suspension: Malfari suspendi
+      unsubscribe: Malaboni
+      username: Uzantnomo
+      web: Ret
+    custom_emojis:
+      copied_msg: Sukcese kreis loka kopio de la emojio
+      copy: Kopi
+      copy_failed_msg: Could not make a local copy of that emoji
+      created_msg: Emojio estas kreita sukcesa!
+      delete: Forigi
+      destroyed_msg: Emojio estas forigis sukcesa!
+      disable: Malebligi
+      disabled_msg: Emojio estas malebligis sukcesa
+      emoji: Emojio
+      enable: Ebligi
+      enabled_msg: Emojio estas ebligis sukcesa
+      image_hint: PNG ĝis 50KB
+      new:
+        title: Aldoni nova kutimo emojio
+      shortcode: Malongakodo
+      shortcode_hint: At least 2 characters, only alphanumeric characters and underscores
+      title: Kutimoj emojioj
+      upload: Alŝuti
+    domain_blocks:
+      add_new: Aldoni novo
+      created_msg: Domajno bloko nun estas procesita
+      destroyed_msg: Domajno bloko estas malfaris
+      domain: Domajno
+      new:
+        create: Krei bloko
+        noop: Nenio
+        silence: Silenti
+        suspend: Suspendi
```

Comment:
https://github.com/tootsuite/mastodon/blob/db33a53ee8a617937ce9f9e1e79a1c5ac00de126/config/locales/en.yml#L149-L153

`severity:` is required.

### screenshot on [CI](https://travis-ci.org/tootsuite/mastodon/jobs/284730573)

![`bundle exec i18n-tasks unused`](https://user-images.githubusercontent.com/12539/31314107-d81c93aa-ac32-11e7-8084-f1ad2f9a4e61.png)

Reply:
:open_mouth:  :+1: